src/plugins/unified_search/public/dataview_picker/sorting_service.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { Direction } from '@elastic/eui';
+import { SortDirection } from '@elastic/eui';
+import type { IStorageWrapper } from '@kbn/kibana-utils-plugin/public';
+import { Storage } from '@kbn/kibana-utils-plugin/public';
+
+const storageKey = 'unified_search_sorting';
+export const ALPHABETICALLY = 'alphabetically';
+
+export interface Sorting {
+  column: typeof ALPHABETICALLY;
+  direction: Direction;
+}
+
+export class SortingService<T = unknown> {
+  public column: Sorting['column'];
+  public direction: Sorting['direction'];
+
+  constructor(
+    private sortingStrategy: Record<Sorting['column'], (arg: T) => string>,
```

Comment:
WDYT about changing from `sortingStrategy` to `sortingStrategies` as it is a record, containing different sorting strategies?